scripts/new_version/new_version.sh

```diff
@@ -37,38 +42,40 @@ new_version_without_revision=$3
 new_version_year=${new_version:1:4}
 new_version_letter=${new_version:5:1}
 
+CURRENT_DIR=$WEBOTS_HOME/scripts/new_version
```

Comment:
```suggestion
CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

This way it will work even if we move it.